pkg/control/synchronizer_test.go

```diff
@@ -73,7 +76,7 @@ func TestSynchronizer_garbageCollect(t *testing.T) {
 				serviceEntry: &mock.SEStore{Result: tt.serviceEntries},
 				client:       &mockIstio{store: make(map[string]*icapi.ServiceEntry)},
 			}
-			s.garbageCollect()
+			s.garbageCollect(context.TODO())
```

Comment:
`TODO` is used to say, we need to use a concrete context but we don't know yet which one.
```suggestion
			s.garbageCollect(context.Background())
```